src/Knp/Menu/MenuItem.php

```diff
@@ -512,11 +520,13 @@ public function actsLikeFirst(): bool
             return true;
         }
 
-        $children = $this->getParent()->getChildren();
-        foreach ($children as $child) {
-            // loop until we find a visible menu. If its this menu, we're first
-            if ($child->isDisplayed()) {
-                return $child->getName() === $this->getName();
+        if (null !== $parent = $this->getParent()) {
```

Comment:
this can be simplified by changing the `isRoot` check above to `null === $this->parent` as well